R/xml_find.R

```diff
@@ -203,3 +207,13 @@ xml_find_lgl.xml_nodeset <- function(x, xpath, ns = xml_ns(x)) {
 xml_find_lgl.xml_missing <- function(x, xpath, ns = xml_ns(x)) {
    logical(0)
 }
+
+# Deprecated functions ----------------------------------------------------
+
+#' @rdname xml_find_all
+#' @usage NULL
+#' @export
+xml_find_one <- function(x, xpath, ns = xml_ns(x)) {
+  .Deprecated("xml_find_first")
```

Comment:
Looks like there are only 6 packages that call `xml_find_one()` on CRAN now, counts are number of search results (files) which contain calls to `xml_find_one()`.

``` r
library(gh)
res <- gh("GET /search/code", q = "user:cran xml_find_one")

library(purrr)
names <- map_chr(res$items, function(x) x$repository$name)
table(names)
#> names
#>   docxtractr googlesheets          oai    rversions          rvg 
#>            3            1            1            1           16 
#>      svglite         xml2 
#>            4            4
```